templates/patterns/banner/pattern-banner--variant-image_shade.html.twig

```diff
@@ -0,0 +1,20 @@
+{#
+/**
+ * @file
+ * Banner component.
+ */
+#}
+{% if centered  %}
+  {% set attributes = attributes.addClass(['text-center'])%}
+{% endif %}
+{% include '@oe-bcl/bcl-banner/banner.html.twig' with {
```

Comment:
extend pattern-banner.html.twig use block content (or as many blocks as you need) to do your alterations